### ISSUE TITLE
On Creating Customer get customers_phone_unique Error

### ISSUE THREAD

**Issue by Samyoul** [Samyoul/The-ultimate-repair-system-issues#31]
When entering a customer with no phone number the following error occurs.

```
2017-01-11 17:49:53] production.ERROR: PDOException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'customers_phone_unique' in /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php:479
Stack trace:
#0 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php(479): PDOStatement->execute()
#1 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php(762): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'insert into `cu...', Array)
#2 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php(725): Illuminate\Database\Connection->runQueryCallback('insert into `cu...', Array, Object(Closure))
#3 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php(480): Illuminate\Database\Connection->run('insert into `cu...', Array, Object(Closure))
#4 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php(434): Illuminate\Database\Connection->statement('insert into `cu...', Array)
#5 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Query/Processors/Processor.php(32): Illuminate\Database\Connection->insert('insert into `cu...', Array)
#6 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2152): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `cu...', Array, 'id')
#7 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#8 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1464): call_user_func_array(Array, Array)
#9 /var/www/turs/bootstrap/cache/compiled.php(10827): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#10 /var/www/turs/bootstrap/cache/compiled.php(10816): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#11 /var/www/turs/bootstrap/cache/compiled.php(10770): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder))
#12 /var/www/turs/bootstrap/cache/compiled.php(10428): Illuminate\Database\Eloquent\Model->save()
#13 /var/www/turs/app/Http/Controllers/CustomerController.php(47): Illuminate\Database\Eloquent\Model::create(Array)
#14 [internal function]: App\Http\Controllers\CustomerController->store(Object(App\Http\Requests\SaveCustomer))
#15 /var/www/turs/bootstrap/cache/compiled.php(9419): call_user_func_array(Array, Array)
#16 /var/www/turs/bootstrap/cache/compiled.php(9446): Illuminate\Routing\Controller->callAction('store', Array)
#17 /var/www/turs/bootstrap/cache/compiled.php(8500): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(App\Http\Controllers\CustomerController), 'store')
#18 /var/www/turs/bootstrap/cache/compiled.php(8481): Illuminate\Routing\Route->runController()
#19 /var/www/turs/bootstrap/cache/compiled.php(8190): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#20 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 /var/www/turs/app/Http/Middleware/SubdomainManager.php(18): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#22 /var/www/turs/bootstrap/cache/compiled.php(9904): App\Http\Middleware\SubdomainManager->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 /var/www/turs/app/Http/Middleware/DenyAccessForSuperAdmin.php(26): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 /var/www/turs/bootstrap/cache/compiled.php(9904): App\Http\Middleware\DenyAccessForSuperAdmin->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 /var/www/turs/app/Http/Middleware/LogoutIfEmailNotVerified.php(24): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#28 /var/www/turs/bootstrap/cache/compiled.php(9904): App\Http\Middleware\LogoutIfEmailNotVerified->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 /var/www/turs/vendor/laravel/spark/src/Http/Middleware/VerifyUserHasTeam.php(22): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#31 /var/www/turs/bootstrap/cache/compiled.php(9904): Laravel\Spark\Http\Middleware\VerifyUserHasTeam->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 /var/www/turs/vendor/laravel/spark/src/Http/Middleware/CreateFreshApiToken.php(40): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#34 /var/www/turs/bootstrap/cache/compiled.php(9904): Laravel\Spark\Http\Middleware\CreateFreshApiToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 /var/www/turs/bootstrap/cache/compiled.php(10012): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#37 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Routing\Middleware\SubstituteBindings->handle(Object(Illuminate\Http\Request), Object(Closure))
#38 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 /var/www/turs/vendor/laravel/framework/src/Illuminate/Auth/Middleware/Authenticate.php(43): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#40 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Auth\Middleware\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 /var/www/turs/bootstrap/cache/compiled.php(3164): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#43 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#44 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#45 /var/www/turs/bootstrap/cache/compiled.php(13560): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#46 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 /var/www/turs/bootstrap/cache/compiled.php(11983): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#49 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#50 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#51 /var/www/turs/bootstrap/cache/compiled.php(13306): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#52 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#53 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#54 /var/www/turs/bootstrap/cache/compiled.php(13243): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#55 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#56 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#57 /var/www/turs/bootstrap/cache/compiled.php(9889): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#58 /var/www/turs/bootstrap/cache/compiled.php(8191): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#59 /var/www/turs/bootstrap/cache/compiled.php(8182): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#60 /var/www/turs/bootstrap/cache/compiled.php(8173): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#61 /var/www/turs/bootstrap/cache/compiled.php(2471): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#62 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#63 /var/www/turs/bootstrap/cache/compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#64 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#65 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#66 /var/www/turs/bootstrap/cache/compiled.php(9889): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#67 /var/www/turs/bootstrap/cache/compiled.php(2415): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#68 /var/www/turs/bootstrap/cache/compiled.php(2399): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#69 /var/www/turs/public/index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#70 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '' for key 'customers_phone_unique' (SQL: insert into `customers` (`first_name`, `last_name`, `email`, `phone`, `team_id`, `reference`, `updated_at`, `created_at`) values (joe, Blogs, , , 3, 000001, 2017-01-11 17:49:53, 2017-01-11 17:49:53)) in /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php:769
Stack trace:
#0 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php(725): Illuminate\Database\Connection->runQueryCallback('insert into `cu...', Array, Object(Closure))
#1 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php(480): Illuminate\Database\Connection->run('insert into `cu...', Array, Object(Closure))
#2 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Connection.php(434): Illuminate\Database\Connection->statement('insert into `cu...', Array)
#3 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Query/Processors/Processor.php(32): Illuminate\Database\Connection->insert('insert into `cu...', Array)
#4 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(2152): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `cu...', Array, 'id')
#5 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#6 /var/www/turs/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1464): call_user_func_array(Array, Array)
#7 /var/www/turs/bootstrap/cache/compiled.php(10827): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#8 /var/www/turs/bootstrap/cache/compiled.php(10816): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#9 /var/www/turs/bootstrap/cache/compiled.php(10770): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder))
#10 /var/www/turs/bootstrap/cache/compiled.php(10428): Illuminate\Database\Eloquent\Model->save()
#11 /var/www/turs/app/Http/Controllers/CustomerController.php(47): Illuminate\Database\Eloquent\Model::create(Array)
#12 [internal function]: App\Http\Controllers\CustomerController->store(Object(App\Http\Requests\SaveCustomer))
#13 /var/www/turs/bootstrap/cache/compiled.php(9419): call_user_func_array(Array, Array)
#14 /var/www/turs/bootstrap/cache/compiled.php(9446): Illuminate\Routing\Controller->callAction('store', Array)
#15 /var/www/turs/bootstrap/cache/compiled.php(8500): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(App\Http\Controllers\CustomerController), 'store')
#16 /var/www/turs/bootstrap/cache/compiled.php(8481): Illuminate\Routing\Route->runController()
#17 /var/www/turs/bootstrap/cache/compiled.php(8190): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#18 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#19 /var/www/turs/app/Http/Middleware/SubdomainManager.php(18): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 /var/www/turs/bootstrap/cache/compiled.php(9904): App\Http\Middleware\SubdomainManager->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 /var/www/turs/app/Http/Middleware/DenyAccessForSuperAdmin.php(26): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#23 /var/www/turs/bootstrap/cache/compiled.php(9904): App\Http\Middleware\DenyAccessForSuperAdmin->handle(Object(Illuminate\Http\Request), Object(Closure))
#24 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 /var/www/turs/app/Http/Middleware/LogoutIfEmailNotVerified.php(24): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 /var/www/turs/bootstrap/cache/compiled.php(9904): App\Http\Middleware\LogoutIfEmailNotVerified->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 /var/www/turs/vendor/laravel/spark/src/Http/Middleware/VerifyUserHasTeam.php(22): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#29 /var/www/turs/bootstrap/cache/compiled.php(9904): Laravel\Spark\Http\Middleware\VerifyUserHasTeam->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 /var/www/turs/vendor/laravel/spark/src/Http/Middleware/CreateFreshApiToken.php(40): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#32 /var/www/turs/bootstrap/cache/compiled.php(9904): Laravel\Spark\Http\Middleware\CreateFreshApiToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 /var/www/turs/bootstrap/cache/compiled.php(10012): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#35 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Routing\Middleware\SubstituteBindings->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 /var/www/turs/vendor/laravel/framework/src/Illuminate/Auth/Middleware/Authenticate.php(43): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#38 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Auth\Middleware\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 /var/www/turs/bootstrap/cache/compiled.php(3164): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#41 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 /var/www/turs/bootstrap/cache/compiled.php(13560): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#44 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 /var/www/turs/bootstrap/cache/compiled.php(11983): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#47 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#49 /var/www/turs/bootstrap/cache/compiled.php(13306): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#50 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#52 /var/www/turs/bootstrap/cache/compiled.php(13243): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#53 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#54 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#55 /var/www/turs/bootstrap/cache/compiled.php(9889): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#56 /var/www/turs/bootstrap/cache/compiled.php(8191): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#57 /var/www/turs/bootstrap/cache/compiled.php(8182): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#58 /var/www/turs/bootstrap/cache/compiled.php(8173): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#59 /var/www/turs/bootstrap/cache/compiled.php(2471): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#60 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#61 /var/www/turs/bootstrap/cache/compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#62 /var/www/turs/bootstrap/cache/compiled.php(9904): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#63 /var/www/turs/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#64 /var/www/turs/bootstrap/cache/compiled.php(9889): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#65 /var/www/turs/bootstrap/cache/compiled.php(2415): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#66 /var/www/turs/bootstrap/cache/compiled.php(2399): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#67 /var/www/turs/public/index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#68 {main}  
```

**Comment by jcergolj** [comment]
fixed #31

**Comment by Samyoul** [comment]
Just so you know, you can automatically mark issues as solved via commit messages using:

`fix Samyoul/The-ultimate-repair-system-issues#31`
